core/src/main/java/io/questdb/cairo/vm/MemoryPARWImpl.java

```diff
@@ -202,6 +202,13 @@ public final int getInt(long offset) {
         return getInt0(offset);
     }
 
+    public final int getIPv4(long offset) {
```

Comment:
there should be auto-formatting enabled in your intellij

Reply:
Fixed in https://github.com/questdb/questdb/pull/3523/commits/177bce86e12a6d19e8d4c13dd7e97c91f0c09a9e